lib/msal-common/test/authority/Authority.spec.ts

```diff
@@ -907,6 +939,20 @@ describe("Authority.ts Class Unit Tests", () => {
             expect(endpoint).toBe(`${authorityUrl}.well-known/openid-configuration`);
         });
 
+        it("DSTS authority uses v1 well-known endpoint", async () => {
+            const authorityUrl = "https://login.microsoftonline.com/dstsv2/"
+            let endpoint = "";
+            authority = new Authority(authorityUrl, networkInterface, mockStorage, authorityOptions);
+            jest.spyOn(networkInterface, <any>"sendGetRequestAsync").mockImplementation((openIdConfigEndpoint) => {
+                // @ts-ignore
+                endpoint = openIdConfigEndpoint;
+                return DEFAULT_OPENID_CONFIG_RESPONSE;
+            });
+
+            await authority.resolveEndpointsAsync();
+            expect(endpoint).toBe(`${authorityUrl}.well-known/openid-configuration`);
+        });
+
```

Comment:
Is it possible to add a unit test based on the Fiddler trace that was shared? By mocking the HTTP response.

Reply:
I'll give it a shot; I assume I'll just have to translate my manual test to a set of unit tests. Thanks for the added context and feedback!

Reply:
Done!